app/models/card.rb

```diff
@@ -5,14 +5,19 @@ class Card < ActiveRecord::Base
 
   before_validation :set_review_date
 
+  scope :actual_cards, -> { where("review_date <= ?", Date.current) }
+  scope :random_card, -> { order("RANDOM()").first }
+
   private
 
   def check_difference
-    errors.add( I18n.t('error.validation.description.the_same_value') ,I18n.t('error.validation.messages.the_same_value')) if
-        original_text.downcase.present? && translated_text.downcase.present? && original_text.downcase == translated_text.downcase
+    errors.add( I18n.t('error.validation.description.the_same_value') ,
```

Comment:
Space found before comma.<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.